components/profile/Profile.js

```diff
@@ -1,27 +1,131 @@
 import { makeStyles } from '@material-ui/styles';
+import Box from '@material-ui/core/Box';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
 
+import { useAuth } from '../Auth';
+import FirebasePropTypes from '../Firebase/PropTypes';
+import ProfileItemCard from './ProfileItemCard';
+import UserProfileCard from './UserProfileCard';
 import ScaffoldContainer from '../ScaffoldContainer';
 
-const useStyles = makeStyles(() => ({
+const ROW_GAP = 2;
+const COL_GAP = 2;
+
+const useStyles = makeStyles(theme => ({
   root: {
     position: 'relative',
   },
+  subtitle: {
+    display: 'inline-block',
+  },
+  container: {
+    marginTop: theme.spacing(-5),
+  },
+  grid: {
+    display: 'grid',
+    marginTop: theme.spacing(4),
+    gridGap: theme.spacing(ROW_GAP, COL_GAP),
+    gridTemplateAreas: `
+      'H'
+      'C'
+      'L'
+      'R'
+    `,
+
+    [theme.breakpoints.up('md')]: {
+      gridTemplateColumns: '1fr 2fr 1fr',
+      gridTemplateAreas: `
+        '. H .'
+        'L C R'
+      `,
+    },
+
+    /* All Following CSS exists to support IE 11. Keep in sync with the grid CSS above.
+     * Helpful tool: https://autoprefixer.github.io/
+     */
+    '-ms-grid-rows': `
+      auto
+      ${theme.spacing(ROW_GAP)}px
+      auto
+      ${theme.spacing(ROW_GAP)}px
+      auto
+      ${theme.spacing(ROW_GAP)}px
+      auto`,
+  },
+  gridH: { '-ms-grid-row': 1, '-ms-grid-column': 1, 'grid-area': 'H' },
+  gridC: { '-ms-grid-row': 3, '-ms-grid-column': 1, 'grid-area': 'C' },
+  gridL: { '-ms-grid-row': 5, '-ms-grid-column': 1, 'grid-area': 'L' },
+  gridR: { '-ms-grid-row': 7, '-ms-grid-column': 1, 'grid-area': 'R' },
+  [theme.breakpoints.up('md')]: {
+    grid: {
+      '-ms-grid-columns': `1fr ${theme.spacing(COL_GAP)}px 2fr ${theme.spacing(COL_GAP)}px 1fr`,
+      '-ms-grid-rows': `auto ${theme.spacing(ROW_GAP)}px auto`,
+    },
+    gridH: { '-ms-grid-row': 1, '-ms-grid-column': 3 },
+    gridC: { '-ms-grid-row': 3, '-ms-grid-column': 3 },
+    gridL: { '-ms-grid-row': 3, '-ms-grid-column': 1 },
+    gridR: { '-ms-grid-row': 3, '-ms-grid-column': 5 },
+  },
 }));
 
-function Profile() {
+const PROFILE_ITEMS = [
+  {
+    title: 'My Goal',
+    description: 'Edit Your Profile to add your Goal',
+  },
+  {
+    title: 'Education History',
+    description: 'Edit Your Profile to add Education History',
+  },
+  {
+    title: 'Employment Experience',
+    description: 'Edit Your Profile to add Employment Experience',
+  },
+];
+
+const PHONE_QUESTION_ID = 'recDkvWeqvMxahFT2';
+
+function Profile({ allQuestionResponses }) {
   const classes = useStyles();
+  const { user } = useAuth();
+  const questionResponse = allQuestionResponses.find(doc => doc.id === PHONE_QUESTION_ID);
+  const phoneNumber = questionResponse && questionResponse.data().value;
```

Comment:
Is the `phoneNumber` accessible via the `user`? (i.e. do we have to retrieve it from the assessment?)

Reply:
the phoneNumber under user authProfile is generated by the auth provider,  and currently, I didn't see it get populated.